src/main/java/filter/expression/Qualifier.java

```diff
@@ -160,6 +193,15 @@ public static boolean process(IModel model, FilterExpression expr, TurboIssue is
         return Optional.of(allMilestones.size());
     }
 
+    /**
+     * Checks whether a milestone still has open issues or it is not due yet.
+     */
+    private static boolean isRelevantMilestone(TurboMilestone milestone) {
```

Comment:
It's more like `isOpenMilestoneStillOngoing()`, where we consider `ongoing` as:

- not due yet, or
- overdue but still have open issues in it